src/app/content/components/Content.browserspec.tsx

```diff
@@ -9,7 +9,7 @@ const TEST_CASES: { [testCase: string]: (target: Page) => Promise<void> } = {
   Desktop: setDesktopViewport, Mobile: setMobileViewport,
 };
 const EXPECTED_SCROLL_TOPS: { [testCase: string]: number[] } = {
-  Desktop: [242, 90, 122, 242, 365, 668, 761, 1262, 1606],
+  Desktop: [242, 90, 122, 242, 365, 668, 761, 1253, 1597],
```

Comment:
@TomWoodward I don't know why this continues to be an issue but it may be because the content pane max-width is slightly narrower (80px) than before. I assume we want to fix this but it's tricky because of the complex layout. Wondering if we want to fix before or after merging.